roles/installer/vars/main.yml

```diff
@@ -1,7 +1,12 @@
 ---
 # vars file for installer
 pullsecret_file: "{{ dir }}/pull-secret.txt"
-cmd: openshift-baremetal-install
+cmd: >-
```

Comment:
I wonder should we rename this to installer_cmd 
the old name was only used when the command was being extracted but now its getting used in multiple places so something more descriptive might be helpful.

Reply:
Done